src/main/java/org/dependencytrack/parser/dependencytrack/NotificationModelConverter.java

```diff
@@ -182,6 +190,8 @@ private static Optional<com.google.protobuf.Any> convert(final Object subject) {
             return Optional.of(Any.pack(convert(pvi)));
         } else if (subject instanceof final org.dependencytrack.model.Project p) {
             return Optional.of(Any.pack(convert(p)));
+        } else if (subject instanceof final UserPrincipalSubject p) {
+            return Optional.of(Any.pack(convert(p)));
```

Comment:
Looks like we don't need any conversion here since the subject is already a Proto struct.

```suggestion
        } else if (subject instanceof final UserPrincipalSubject p) {
            return Optional.of(Any.pack(p));
```